renovate.json

```diff
@@ -8,8 +8,23 @@
   "packageRules": [
     {
       "groupName": "Style guide maintenance",
-      "packagePatterns": ["^(braid-design-system|(seek-style-guide(-asia)?))$"],
+      "packagePatterns": ["^(braid-design-system|(seek(-asia)?-style-guide))$"],
       "schedule": ["before 8am on the first day of the month"]
+    },
+    {
+      "groupName": "Webpack maintenance",
+      "packagePatterns": ["webpack|(-loader$)|autoprefixer|less|css|svg"],
+      "schedule": ["before 8am on Thursday"]
+    },
+    {
+      "groupName": "Babel maintenance",
+      "packagePatterns": ["^@?babel[/-]"],
+      "schedule": ["before 8am on Thursday"]
+    },
+    {
+      "groupName": "Tooling maintenance",
+      "packagePatterns": ["jest|puppeteer|tslint|eslint|prettier"],
```

Comment:
I'm less interested in prettier. Given the requirement for consumers to run commands to update when prettier updates I'm more happy for it to go stale and get major leaps forward.

If we want it, I'd suggest adding it as a seperate group and scheduling it every 1-3 months.

Reply:
Yep im happy with that. Will remove prettier from this for now.